src/amo/components/AddonBadges/index.js

```diff
@@ -30,24 +35,23 @@ export const AddonBadgesBase = (props: Props) => {
     }
   };
 
-  const isIncompatible = addon && addon.type === ADDON_TYPE_EXTENSION &&
-    !addon.isWebExtension;
+  const isIncompatible = isQuantumCompatible({ addon }) === false;
```

Comment:
I think it would be safer to do this as:

```js
const isIncompatible = addon.type === ADDON_TYPE_EXTENSION &&
  isQuantumCompatible({ addon }) === false;
```

It's hurting my brain a little bit but I'm worried because [the compatibility object patch](https://github.com/mozilla/addons-frontend/commit/3cecbed3269f715099a6c8c69430bd9da6d6b2da) has not landed in prod yet which means when we cherry pick it, this function may be getting compatibility objects that cannot be trusted.

Reply:
but then we'll loose the possibility to test other add-on types?

Reply:
nevermind, let's play safe and do something better, later.